src/illuminator/builder/model.py

```diff
@@ -70,7 +71,7 @@ def simulator_meta(self) -> dict:
         meta = {
             'type': self.simulator_type.value,
             'models': {
-                'Model': {
+                self.model_type : {
```

Comment:
This specific line changed due to the following error:
`AttributeError: Model factory for "Adder1-0" has no model and no function "Adder".`

This is caused by this exact line.
When self.meta['models'] is called, it searches for the appropriate model name. Leaving the name as 'Model' causes issues